content/applications/finance/fiscal_localizations/spain.rst

```diff
@@ -2,34 +2,180 @@
+you can see the traceability of other generated documents related to the invoice (ex. if the invoice
+should also be sent through the **SII**, it will appear here).
+
+.. image:: spain/chatter.png
+   :alt: XML file in the chatter of the invoice.
+
+.. note::
+   The TBAI **QR code** is on the PDF of the invoice.
+
+   .. image:: spain/qr-code.png
+      :alt: QR code of the TicketBAI on the invoice.
+
+FACe
+====
+
+Before configurating the :abbr:`FACe (Punto General de Entrada de Facturas Electrónicas)` system,
+make sure to have the :guilabel:`Spain - Facturae EDI (l10n_es_edi_facturae)` module and other
+related **Facturae EDI** modules installed. If not installed, you can install by going to **Apps**
+and searching for `Facturae EDI`.
+
+.. image:: spain/facturae-edi.png
+   :alt: FACe module in Odoo.
+
+To enable FACe, first set the :guilabel:`Country` and :guilabel:`Tax ID` of your company under
+:menuselection:`Settings --> General Settings --> My Company` in the :guilabel:`Companies` section,
+and clicking :guilabel:`Update Info`. Next, add the :guilabel:`Facturae signature certificate` by
+clicking :guilabel:`Add a line` and uploading the certificate provided by the tax agency as well as
+entering the password provided by the agency.
+
+.. image:: spain/facturae-certificate.png
+   :alt: Facturae certificate field in the company form.
+
+Use case
+--------
+
+Once you have :doc:`created <../../finance/accounting/customer_invoices>` an invoice and confirmed
+it, click :guilabel:`Send and Print`. Make sure :guilabel:`Generate Facturae edi file` is enabled,
+and click :guilabel:`Send & Print` again. Once sent, the generated XML file is available in the
+**chatter**.
+
+.. image:: spain/facturae-checkbox.png
+   :alt: Facturae EDI file checkbox when sending an invoice.
+
+.. warning:
+   The file is **NOT** automatically sent. You have to send it yourself manually.
+
+.. tip::
+   You can send **FACe** XML files in batch through `here <https://www.facturae.gob.es/formato/Paginas/descarga-aplicacion-escritorio.aspx>`_.
+
+.. image:: spain/facturae-chatter.png
+   :alt: XML file generated found in the chatter of the invoice.
+
+Administrative centers
+----------------------
+
+In order for **FACe** to work with **administrative centers**, the invoice *must* include specific
+data about the centers.
+
+.. tip::
+   Make sure to have the :guilabel:`Spain - Facturae EDI - Administrative Centers Patch
+   (l10n_es_edi_facturae_adm_centers)` module installed.
+
+To add **admnistrative centers**, create a new **contact** to add to the **partner** company. Select
+:guilabel:`FACe Center` as **type**, assign one or more **role(s)** to that contact, and
+:guilabel:`Save`. The **three** roles usually required are:
+
+- Órgano gestor: :guilabel:`Receptor` (Receiver);
+- Unidad tramitadora: :guilabel:`Pagador` (Payer);
+- Oficina contable: :guilabel:`Fiscal` (Fiscal).
```

Comment:
Why these roles? Can we have a bit of explanation?

Reply:
The info in this doc comes straight from Spanish BAs in Valencia and is the document they use, it's not included in it and goes beyond my understanding. I believe users dealing with their accounting would know what the roles correspond to. IMO I do not believe it is necessary to explain them